tic-tac-toe.js

```diff
@@ -0,0 +1,13 @@
+const x = 1
+const o = 0
+let grid = {
+    tl: null,
+    tm: null,
+    tr: null,
+    ml: null,
+    mm: null,
+    mr: null,
+    bl: null,
+    bm: null,
+    br: null,
+};
```

Comment:
If you have any spare time, a stretch goal for your would be to go through the setup for eslint so it would call this type of thing inside your code editor

Reply:
Didn't have time but will get to it.